lib/crypto/butil.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
Ok thanks, looks like the `bufferize` in fxa-auth-server works on Objects, so I'm gonna keep that one for now.